### DISCUSSION TITLE
Business usage of the komoot hosted  photon API?

### DISCUSSION THREAD

**Discussion by juergenhah** [komoot/photon#598] [General]
For some business customers we host a photon service which is used in several commercial geo applications. 

The [https://photon.komoot.io](https://photon.komoot.io) Homepage states that the usage of the photon hosted by komoot can be used free of charge.

We ask ourselves, if we switch to the photon hosted by komoot.

As we operate the photon for our customers on service levels we need to know following things:

- The photon Homepage states that "extensive usage“ will be throttled. Which amount of requests do you consider as extensive? Our services typically serve 300 requests a day and 5000 maximum on a day. Would you throttle this amount? If so, would we receive a notification before, e.g an email if we reach a threshold? It is important to us to guarantee this service for our customers.

- Would it be possible to have a kind of „Business Plan“ for the usage of your photon? If it would we a fair price/usage condition we are willing to pay for the usage. 

Thanks in advance for your response and we are happy to discuss things in detail.

**Comment by lonvia** [comment]
The site at https://photon.komoot.io is a demo site for the Photon software. We don't give any uptime guarantees and reserve the right to roll out new (and sometimes experimental) features that might break your use of the service. Consequently, it is not a good idea to send your business clients over there. There is no hard definition on "extensive usage". We throttle usage as required to protect the availability of the service for all. As a rule of thumb: if you have to ask about the usage limits, you are likely better off running your own instance.

Neither Komoot nor the maintainers of Photon provide a commercial API for Photon and I'm not aware of such an offer elsewhere. However, there are a couple of companies that provide geocoding services, usually with their own custom API but based on a similar dataset and thus with similar quality and coverage. Have a look at this [list in the OSM wiki](https://wiki.openstreetmap.org/wiki/Nominatim#Alternatives_.2F_Third-party_providers) for some examples.
